.github/workflows/e2e-polybft-test.yml

```diff
@@ -41,6 +41,20 @@ jobs:
         uses: milos-ethernal/cardano-node-action@5a3bce7266cc1500d96e8fa63b3bb97135988482
       - name: Install ogmios
         uses: milos-ethernal/ogmios-action@13e2be0b00da80f26bb8583fa129bd7c08856a49
+      - name: Checkout apex-bridge repo
+        uses: actions/checkout@v4.1.1
+        with:
+          repository: Ethernal-Tech/apex-bridge
+          ref: main
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      - name: Build apex-bridge
+        run: |
+          go build
+          sudo mv apex-bridge /usr/local/bin/
+      - name: Verify apex-bridge executable
+        run: apex-bridge --help
+      - name: Checkout blade
+        uses: actions/checkout@v4.1.1
```

Comment:
Reminder: This workflow will never be triggered on the `feature/apex-bridge` branch. Therefore, when we include it in the `ApexBridge CI`, don't forget to add the same `secrets` block as in the `e2e-apex-test.yml` file to ensure it works properly.